app/views/PlatformInitiativesListPage.scala.html

```diff
@@ -0,0 +1,137 @@
+        <div class="col-xs-1 padding-reset-right">
+            <label for="search">Search:</label>
+        </div>
+        <div class="col-xs-5">
+            <input class="search form-control" id="search" type="text" name="initiativeName"
+                   value='@form("initiativeName").value'>
+        </div>
+        <div class="right" style="line-height:2.2em;">
+            <a href="?chart=true&progress=false">Pie chart</a> |
+            <a href="?chart=false&progress=true">Progress bar</a>
+        </div>
+        <br><br><br>
+        <table class="table table-striped">
+            <tbody class="list">
+            @initiatives.zipWithIndex.map{case (initiative, i) =>
+                @if(initiative.targetProgress != 0){
+                <tr class="col-sm-6">
+                    <td>
+                        @if(displayChart && !displayProgress){
+                        <div class="initiativeName" style="width:500px;">
+                            <h3> @initiative.initiativeName </h3><br \>
+                        </div>
+                            <div id="chart_div_@initiative.initiativeName"></div>
+                        <br>
+                        <div style="max-width:500px; height:100px; word-wrap:break-word;">
+                            @Html(platformInitiativesService.markdown(initiative.initiativeDescription))
+                        </div>
+                        }else if(!displayChart && displayProgress){
+                            <div class="initiativeName" style="width:500px;">
+                                <h3> @initiative.initiativeName </h3>
+                                <progress class="progress" value="@initiative.currentProgress"
+                                          max="@initiative.targetProgress">
+                                    @initiative.currentProgress/@initiative.targetProgress
+                                </progress>
+                            </div>
+                        }else{
+                        <div class="initiativeName" style="width:500px;">
+                            <h3> @initiative.initiativeName </h3><br \>
+                        </div>
+                        <div id="chart_div_@initiative.initiativeName"></div>
+                        <br>
+                        <div style="max-width:500px; height:100px; word-wrap:break-word;">
+                            @Html(platformInitiativesService.markdown(initiative.initiativeDescription))
+                        </div>
+                        <progress class="progress" value="@initiative.currentProgress"
+                                  max="@initiative.targetProgress"></progress>
+                        }
+                    </td>
+                </tr>
+                    }
+                }
+            </tbody>
+        </table>
+    </form>
+</div>
+}
+
+<script type="text/javascript">
+        google.charts.load('current', {'packages': ['corechart']});
```

Comment:
CLARIFICATION: What's the licence for using google charts? Can we just use it?

Reply:
I believe it's Apache 2.0 - It's also used elsewhere on the Catalogue (PrototypeInfoPage, BobbyRulesTrendPage)